apis/r/R/write_seurat.R

```diff
@@ -577,6 +578,130 @@ write_soma.Seurat <- function(
       immediate. = TRUE
     )
   }
-  experiment$close()
+
+  # Add extra Seurat data
+  experiment$add_new_collection(
+    object = SOMACollectionCreate(
+      uri = file_path(experiment$uri, 'uns'),
+      platform_config = platform_config,
+      tiledbsoma_ctx = tiledbsoma_ctx
+    ),
+    key = 'uns'
+  )
+
+  # Write command logs
+  for (cmd in SeuratObject::Command(x)) {
+    spdl::info("Adding command log {}", sQuote(cmd))
+    write_soma(
+      x = x[[cmd]],
+      uri = cmd,
+      soma_parent = experiment$get('uns'),
+      platform_config = platform_config,
+      tiledbsoma_ctx = tiledbsoma_ctx
+    )
+  }
+
   return(experiment$uri)
 }
+
+#' @rdname write_soma_seurat_sub
+#'
+#' @section Writing \code{\link[SeuratObject]{SeuratCommand}s}:
+#' \pkg{Seurat} \link[SeuratObject:SeuratCommand]{command logs} are written out
+#' as \link[tiledbsoma:SOMADataFrame]{data frames} to the
+#' \dQuote{\code{seurat_commands}} group of a
+#' \link[tiledbsoma:SOMACollection]{collection}
+#'
+#' @method write_soma SeuratCommand
+#' @export
+#'
+write_soma.SeuratCommand <- function(
+  x,
+  uri = NULL,
+  soma_parent,
+  ...,
+  platform_config = NULL,
+  tiledbsoma_ctx = NULL,
+  relative = TRUE
+) {
+  check_package('SeuratObject', version = .MINIMUM_SEURAT_VERSION())
+  check_package('jsonlite')
```

Comment:
Please add jsonlite to Suggests: in `DESCRIPTION`

Reply:
🤦 Can't believe I forgot that, thanks for catching. Added in c5a9bc77658c6a05033bf36b2f5c699f1ebd9604